src/components/kubectl/dashboard.ts

```diff
@@ -85,7 +85,8 @@ async function openDashboardForAKSCluster (kubectl: Kubectl): Promise<void> {
     const boundPort = await portForwardToPod(kubectl, dashboardPod, portMapping, 'kube-system');
 
     setTimeout(() => {
-        opn(`http://localhost:${boundPort[0]}`);
+        const openExternal: any = (<any>vscode.env).open ? (<any>vscode.env).open : opn;
```

Comment:
Suggest we put this in a utility module somewhere, so that the API munging logic doesn't pollute the call sites.

Reply:
I noticed our example was out of date - the API name got updated to "openExternal", so this should be `(<any>vscode.env).openExternal`.  Sorry about that!

Reply:
@itowlson extracted to a utility module.

@Chuxel updated.